x-pack/test/security_solution_api_integration/test_suites/investigation/timeline/security_and_spaces/configs/ess.basic.config.ts

```diff
@@ -20,7 +20,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([])}`,
       ],
     },
-    testFiles: [require.resolve('../ess/basic')],
+    testFiles: [require.resolve('../tests/basic')],
```

Comment:
@angorayc , are these event running? I could not find a way to run these locally. 

currently, in my ticket I was working towards enabling them as a part of MKI. We you able to run them?

Reply:
In this branch I can see them is part of `.buildkite/ftr_security_stateful_configs.yml`
  - x-pack/test/security_solution_api_integration/test_suites/investigation/timeline/security_and_spaces/configs/ess.basic.config.ts
  - x-pack/test/security_solution_api_integration/test_suites/investigation/timeline/security_and_spaces/configs/ess.trial.config.ts

I had no problem to run them locally, it just tests in ess.trial.config.ts are skipped.

```
cd x-pack

node scripts/functional_tests_server.js --config ./test/security_solution_api_integration/test_suites/investigation/timeline/security_and_spaces/configs/ess.trial.config.ts
```

Once started, in another terminal
```
node ../scripts/functional_test_runner --config=test/security_solution_api_integration/test_suites/investigation/timeline/security_and_spaces/configs/ess.trial.config.ts
```

Reply:
Ohh okay. I will checkout `.buildkite/ftr_security_stateful_configs.yml`

>I had no problem to run them locally, it just tests in ess.trial.config.ts are skipped

Yes I was working on unskipping those but I was also reorganizing them based on licence similar to other folders so that they can also run in MKI. I need to check how re-org impacts `.buildkite/ftr_security_stateful_configs.yml`

Thanks for the info @angorayc .